api/sft/db.go

```diff
@@ -90,54 +89,91 @@ func (db *Db) DisableFeature(ctx context.Context, toggleId uuid.UUID) error {
 	return nil
 }
 
-func (db *Db) EnableFeature(ctx context.Context, toggleId uuid.UUID) error {
-	sql := `update sft.feature_toggles set enabled = true where id = $1`
+func (db *Db) GetAllToggles(ctx context.Context) ([]*Toggle, error) {
+	sql := `select * from sft.feature_toggles`
```

Comment:
I would prefer to specifically select here - that means if we add another column to the DB it won't break your scan into a struct if the struct isn't yet updated. I.e. running migrations to update the db is less likely to break the code.

Reply:
Awesome, done